### ISSUE TITLE
Añadir al modelo de proyecto un status

### ISSUE THREAD

**Issue by pablovg2** [socialinn/socialtocrowd#113]
Añadir un control para poder activar desactivar los proyectos.  Status: activo, borrador, desactivado, finalizado

**Comment by alesegdia** [comment]
En cuanto a ésto, habría que aclarar que pasa en cada estado. Voy a poner lo que yo creo que sería para cada uno:

* **activo**: público
* **borrador**: no público, solo visible para la organización
* **desactivado**: no sé si desactivado sería más o menos igual que borrador a efectos prácticos. No público, solo visible para la organización.
* **finalizado**: público, pero marcado de alguna forma como que está finalizado, no permitiendo hacer nada en el proyecto, solo ver resultados.

**Comment by pablovg2** [comment]
Sí, el finalizado quizás se podría concretar en que ya no tendría donaciones. El resto de funcionalidades sí se podrían mantener (editar el proyecto, añadir imagenes, etc...)
